ovn_cluster.sh

```diff
@@ -231,6 +261,34 @@ function wait-containers() {
     done
 }
 
+# Starts OVN dbs RAFT cluster on ovn-central-1, ovn-central-2 and ovn-central-3
+# containers.
+function start-db-cluster() {
+    ${RUNC_CMD} exec ${CENTRAL_NAME}-1 ${OVNCTL_PATH} --db-nb-addr=170.168.0.2 --db-nb-create-insecure-remote=yes  \
+--db-sb-addr=170.168.0.2 --db-sb-create-insecure-remote=yes --db-nb-cluster-local-addr=170.168.0.2 \
+--db-sb-cluster-local-addr=170.168.0.2 start_ovsdb
+
+    ${RUNC_CMD} exec ${CENTRAL_NAME}-2 ${OVNCTL_PATH} --db-nb-addr=170.168.0.3 --db-nb-create-insecure-remote=yes  \
+--db-sb-addr=170.168.0.3 --db-sb-create-insecure-remote=yes \
+--db-nb-cluster-local-addr=170.168.0.3 --db-nb-cluster-remote-addr=170.168.0.2 \
+--db-sb-cluster-local-addr=170.168.0.3 --db-sb-cluster-remote-addr=170.168.0.2 start_ovsdb
+
+    ${RUNC_CMD} exec ${CENTRAL_NAME}-2 ${OVNCTL_PATH} --db-nb-addr=170.168.0.4 --db-nb-create-insecure-remote=yes  \
```

Comment:
@numansiddique I guess this should be `${CENTRAL_NAME}-3` if I'm reading it correctly, right?

Reply:
Oops. Indeed.

Reply:
Fixed it.